vnext/Shared/OInstance.cpp

```diff
@@ -649,6 +650,12 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
       []() { return std::make_unique<PlatformConstantsModule>(); },
       nativeQueue));
 
+  modules.push_back(std::make_unique<CxxNativeModule>(
+      m_innerInstance,
+      StatusBarManagerModule::Name,
+      []() { return std::make_unique<StatusBarManagerModule>(); },
```

Comment:
Intentionally not noexcept. I know this has come up before.

When we call a throwing function (e.g. `make_unique`) in something noexcept, we need to generate the implicit catch and terminate. This paradoxically means declaring this as noexcept actually increases binary overhead, since our caller isn't noexcept either, and won't benefit from being able to omit anything itself.